eduNEXT/xblock-limesurvey
# feat: add api credentials in django settings

### Description
- This PR adds the credentials _(username and password)_ to authenticate to the LimeSurvey API as a Django settings. The `session_key` is renewed every time it expires.
- The **session key** was removed from the XBlock configuration in Studio.

### How to test
1. Configure a survey as in [this PR](https://github.com/eduNEXT/xblock-limesurvey/pull/3).
2. In Studio add/edit the unit with the XBlock, and edit the configuration changing the Survey ID if necessary.
3. From the LMS access to the unit with the XBlock.
4. The survey should load normally.
5. As the `session_key` expires every 2 hours, it's recommended to access the survey again after this time, and verify that the survey loads normally.
